src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -182,6 +184,98 @@ private function getEventFromTypeDeclaration(ContainerBuilder $container, string
 
         return $name;
     }
+
+    private function computeBeforeAfterPriority(ContainerBuilder $container): void
+    {
+        $listeners = [];
+        foreach ($container->findTaggedServiceIds('kernel.event_listener', true) as $id => $events) {
+            foreach ($events as $event) {
+                if (!isset($event['event'])) {
+                    if ($container->getDefinition($id)->hasTag('kernel.event_subscriber')) {
+                        continue;
+                    }
+
+                    $event['method'] ??= '__invoke';
+                    $event['event'] = $this->getEventFromTypeDeclaration($container, $id, $event['method']);
+                }
```

Comment:
I've stolen this piece of code [few lines above](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php#L74-L81)

I'm wondering if we should extract a private method, and use as well here the `$aliases` variables :thinking: